assets/css/layout/_footer.scss

```diff
@@ -0,0 +1,242 @@
+// Footer styles.
+
+footer {
+  background: $navy-light;
+  color: $white;
+  margin-top: rem(30);
+  padding: rem(16) rem(32);
+
+  .netlify-notice {
+    padding-top: rem(32);
+    text-align: center;
+
+    a {
+      color: $white;
+      font-weight: $font-weight-normal;
+
+      &:hover,
+      &:focus {
+        color: $navy-light;
+      }
+    }
+  }
+
+  .footer-content {
+    align-items: baseline;
+    display: flex;
+    flex-direction: column;
+    font-size: rem(16);
+    line-height: rem(29);
+    margin: auto;
+    max-width: $max-width-margins;
+
+    h2 {
+      color: $white;
+      font-size: rem(16);
+      font-weight: bold;
+    }
+
+    .funders {
+      p {
+        margin: 0;
+      }
+
+      a {
+        color: $white;
+      }
+
+      svg {
+        padding: rem(13) rem(13) rem(13) 0;
+        width: rem(144);
+      }
+    }
+
+    .contact-info {
+      width: 100%;
+
+      address {
+        font-style: normal;
+      }
+
+      a {
+        color: $white;
+        font-weight: $font-weight-normal;
+
+        svg {
+          padding: rem(32) rem(16) rem(16) 0;
+          width: rem(240);
+        }
+      }
+
+      .logo-ocadu-wrapper {
+        text-align: center;
+      }
+
+      .logo-ocadu {
+        a svg {
+          padding: rem(16);
+          width: rem(128);
+        }
+      }
+    }
+
+    .footer-nav {
+      display: none;
+    }
+
+    .footer-nav,
+    .contact-info,
+    .funders {
+      a:hover,
+      a:focus {
+        color: $navy-light;
```

Comment:
Comparing the hover/focus states of the footer navigation to the header navigation, I wonder if using a larger box shadow might be good. I find the links look tiny surrounded by a lot of blue space.

### Header

<img width="340" alt="Screen Shot 2020-04-03 at 9 25 33 AM" src="https://user-images.githubusercontent.com/605361/78377636-19170900-758d-11ea-9e2d-c19a2aeca44a.png">

### Footer

<img width="360" alt="Screen Shot 2020-04-03 at 9 25 15 AM" src="https://user-images.githubusercontent.com/605361/78377660-1f0cea00-758d-11ea-94c4-925ee865a8ff.png">